### ISSUE TITLE
Query with multiple where is not properly highlighted

### ISSUE THREAD

**Issue by vladshcherbin** [scriptcoded/sql-highlight#8]
👋 

Hey, I've got a query with multiple `where` and `limit`:

```sql
select "users".* from "users" where ("username" = 'test' or "email" = 'test') and "is_admin" = true and "is_banned" = false limit 1
```

It's not properly highlighted, `and` and `limit` words:

![image](https://user-images.githubusercontent.com/6711845/80007986-bb147c00-84cf-11ea-98d4-595ee47001c3.png)

v3.1.0

**Comment by scriptcoded** [comment]
Thanks for the report!

Resolved with 7cae738585ab16769324a6b11c510fbbc6e2fc69. I'll make sure to release this along with at least #5 later today.

**Comment by vladshcherbin** [comment]
@scriptcoded thank you a lot for looking through the issues ❤️

**Comment by scriptcoded** [comment]
@vladshcherbin Thank you just as much for actually taking the time to report them!

**Comment by scriptcoded** [comment]
Released with 3.2.0
